kgforge/core/conversions/dataframe.py

```diff
@@ -63,26 +63,32 @@ def _from_dataframe(row: Series, na: Union[Any, List[Any]], nesting: str) -> Res
     data = deflatten(items, nesting)
     return from_json(data, None)
 
-
 def deflatten(items: List[Tuple[str, Any]], sep: str) -> Dict:
```

Comment:
docstring?

Reply:
well, no other function in the file has them, but still, I added it